testsuite/features/step_definitions/common_steps.rb

```diff
@@ -755,6 +755,18 @@
   node.run("apt-get update")
 end
 
+When(/^I enable repositories before installing Salt on this "([^"]*)"$/) do |host|
+  step %(I enable repository "tools_additional_repo" on this "#{host}")
+end
+
+When(/^I install the latest Salt on this "([^"]*)"$/) do |host|
+  step %(I run "zypper install -y salt-minion" on "#{host}")
```

Comment:
1 - Why not `I install packages "..." on this "..."`

2 - Also  specify "salt" package

Reply:
Actually, we have the following step in `salt_steps.rb`:
```
When(/^I install Salt packages from "(.*?)"$/) do |host|
  target = get_target(host)
  if %w[sle_minion ssh_minion sle_client sle_migrated_minion].include?(host)
    target.run("test -e /usr/bin/zypper && zypper --non-interactive install -y salt salt-minion", false)
.
.
.
```
We don't need to add a new step: we could use teh existent one.

Reply:
The word "from" on the current step, doesn't look coherent :thinking: